src/pages/CreateIssuePage/__tests__/CreateIssuePage.test.tsx

```diff
@@ -0,0 +1,80 @@
+import { cleanup, act } from "@testing-library/react";
```

Comment:
Are you testing the functionality of this page?

Reply:
Yes. I tested the IssueForm component by passing mocked onSubmit function to it